### ISSUE TITLE
LaTeX support

### ISSUE THREAD

**Issue by bchong95** [illinois-cs241/illinois-cs241.github.io#33]
It would be nice if we could add LaTeX support to the docs. 

From a user perspective it should just be like injecting it into the markdown.

I know that there a lot of javascript libraries that support his already

**Comment by earwig** [comment]
Pretty simple, [just](http://docs.mathjax.org/en/latest/start.html)

```html
<script type="text/javascript" async src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML"></script>
```

[then](http://kramdown.gettalong.org/syntax.html#math-blocks)

```
$$
f(x) = \sqrt{x}
$$
```

**Comment by bchong95** [comment]
Resolved in cfd132f06ca7ff90a5864154d13ea5664f934b94
